frontend/package.json

```diff
@@ -54,7 +55,6 @@
     "eslint-plugin-react": "^7.18.0",
     "react-hot-loader": "^4.8.8",
     "redux-mock-store": "^1.5.4",
-    "styled-components": "^5.0.0",
```

Comment:
Why remove styled-components? I know @MittBrukernavn would like to use it

Reply:
it was moved from devDependencies to dependencies